.editorconfig

```diff
@@ -0,0 +1,23 @@
+# EditorConfig is awesome: https://github.com/editorconfig/editorconfig
+
+# Top-most EditorConfig file
+root = true
+
+[*]
+# Set default charset to utf-8
+charset = utf-8
+# Set default indentation to spaces
+indent_style = space
+# Linux-style newlines with a newline ending every file
+end_of_line = lf
+insert_final_newline = true
+# Remove whitespace characters preceding newline characters
+trim_trailing_whitespace = true
+
+# Two space indentation for JavaScript files
+[*.{js,json}]
+indent_size = 2
+
+# Disable trimming trailing whitespaces so that double space newlines work
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
out of curiosity; Does prettier respect this rule?

---

isn't it simpler to read the intention of the author  by just using/forcing the use of `<br/>`?

`  ` is hidden and hard to spot when reviewing/reading/updating docs
with `<br />`, you at least see the intent and the cognitive overload is out of the way

Reply:
I think prettier does not touch this as the white space would be between words. Maybe a br would be easier to read but isnt possible in all places in md files.